wrap/src/components/WrapForm/index.tsx

```diff
@@ -79,7 +79,10 @@ export const WrapForm: FC = () => {
 
       navigate(`/wrap/tx/${txHash}?amount=${value}&action=${formType}`)
     } catch (ex) {
-      setError((ex as Error)?.message || JSON.stringify(ex))
+      // E.g. ex.shortMessage: 'User rejected the request.'
+      // ex.message: 'User rejected the request.\n\nRequest Arguments:\n  from: ...'
+      // ex.cause.cause.code: 4001
+      setError((ex as BaseError)?.shortMessage || (ex as Error)?.message || JSON.stringify(ex))
```

Comment:
Isn't default shortMessage always provided https://github.com/wevm/viem/blob/main/src/errors/base.ts#L60 or are we catching not viewm errors here as well and not all will have shortMsg?

Reply:
I didn't check if non-viem error can happen. Previous code assumed so - message might be missing